### DISCUSSION TITLE
Dragoon combo and ninja combo

### DISCUSSION THREAD

**Discussion by PedroRamalho12** [PrincessRTFM/XIVComboPlugin#44] [Ideas]
Did you realize that drg combo does not proc vorpal blade after the full ending with raiden thrust, it should proc vorpal and disenbowel, and it only procs disenbowel. on the the combo.
and with ninja could you add an option with the timer on the buff? instead of auto proc armor crush to reset timer below 30 sec I wanted to choose the timer on the proc just like u did on warrior buff.

Keep the good work !

**Comment by VariableVixen** [comment]
Moved to issue (#45)
